source/reference/configuration-options.txt

```diff
@@ -3201,10 +3201,19 @@ Key Management Configuration Options
    authenticating the user. :binary:`~bin.mongod` or :binary:`~bin.mongos` does not check the remaining documents
    in the array.
    
-   If the given document does not match the provided authentication name, or
-   the transformation described by the document fails, :binary:`~bin.mongod` or :binary:`~bin.mongos` continues
-   through the list of documents to find additional matches. If no matches are
-   found in any document, :binary:`~bin.mongod` or :binary:`~bin.mongos` returns an error.
+   If the given document does not match the provided authentication
```

Comment:
Also, need to update
https://docs.mongodb.com/manual/reference/program/mongod/#cmdoption-mongod-ldapusertodnmapping
https://docs.mongodb.com/manual/reference/program/mongos/#cmdoption-mongos-ldapusertodnmapping

Reply:
updated plus also updated https://docs.mongodb.com/master/reference/program/mongoldap/#cmdoption-mongoldap-ldapusertodnmapping